QRHunter/app/src/main/java/com/example/qrhunter/data/repository/QRCodeRepository.java

```diff
@@ -26,32 +31,62 @@ public class QRCodeRepository extends DataRepository {
      * @param playerId The id of the player
      * @param qrCode   The id of the qrCode
      */
-    public void addQRCodeToPlayer(QRCode qrCode, String playerId) {
+    public void addQRCodeToPlayer(QRCode qrCode, String playerId, byte[] savedPhoto) {
         PlayerRepository playerRepository = new PlayerRepository();
 
         // Check whether qr code exists.
         doesQRCodeExist(qrCode, existingQRCode -> {
             if (existingQRCode == null) {
                 // If qr code doesn't exist, add a new document to Firestore
-                Map<String, Object> qrCodeHashMap = QRCodeUtil.convertQRCodeToHashmap(qrCode);
+                String photoPath;
+                ArrayList<String> photos = new ArrayList<>();
+                String qrCodeId = qrCode.getId();
+                if (savedPhoto == null) {
+                    photoPath = null;
+                }
+                else {
+                    photoPath = "photos/" + qrCodeId + "/" + playerId + ".png";
```

Comment:
I think storing it as jpg would fit the issue #14 better, since jpg are usually smaller in size compared to png

Reply:
Okay